materials/knitr-examp.Rmd

```diff
@@ -19,8 +19,8 @@ library(scales)
 Data is from the [Portal project teaching database](http://figshare.com/articles/Portal_Project_Teaching_Database/1314459) published on Figshare. We need the surveys table for our analysis:
 
 ```{r, cache=TRUE}
-# Setting cache=TRUE lets you re-use the results of this code chunk instead of
-# re-calculating or re-downloading it from scratch.
+# Setting cache=TRUE lets you re-use the results of this code chunk in other code
+# chunks instead of re-calculating or re-downloading it each time.
```

Comment:
I'm not sure this change exactly right. Code chunks share state automatically. I'm pretty sure that `cache` is primarily operating between knits, not within a single knit.